test/conformance/walk.go

```diff
@@ -85,12 +87,17 @@ func (v *visitor) convertToConformanceData(at *ast.BasicLit) {
 	cd.Description = ""
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
-		if strings.HasPrefix(line, "Testname:") {
+		if ok, _ := regexp.MatchString("^\\s*Test[N|n]ame\\s*:", line); ok {
```

Comment:
Why the mixed caps here, where do we have discrepancies in our codebase?

Reply:
yes please

Reply:
Fixed. Please review.